covid_xprize/standard_predictor/tests/test_xprize_predictor.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
+
+import os
+import unittest
+import urllib.request
+
+import pandas as pd
+
+from covid_xprize.standard_predictor.xprize_predictor import XPrizePredictor
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+EXAMPLE_INPUT_FILE = os.path.join(ROOT_DIR, "../../validation/data/2020-09-30_historical_ip.csv")
```

Comment:
This predictor has fixed weights and data, so use them directly, i.e., no need for FIXTURES.